packages/teams-js/src/public/tasks.ts

```diff
@@ -85,4 +92,10 @@ export namespace tasks {
       return TaskModuleDimension.Small;
     }
   }
+
+  export function getDefaultSizeIfNotProvided(taskInfo: TaskInfo): TaskInfo {
```

Comment:
Same question I left on the other PR: we had mentioned in the doc we showed to partners that we would default to Small. Why the change?

Reply:
It was not intentional. Sorry, I missed it from the doc. Will change to small